lib/pbench/test/functional/server/test_datasets.py

```diff
@@ -72,10 +72,17 @@ def test_upload_all(self, server_client: PbenchServerClient, login_user):
             assert (
                 response.status_code == HTTPStatus.CREATED
             ), f"upload returned unexpected status {response.status_code}, {response.text} ({t})"
+            benchmark = server_client.get_metadata(md5, ["server.benchmark"])[
+                "server.benchmark"
+            ]
```

Comment:
I considered that, because I don't like the wrapping: but I didn't really like the temporary variable, too, and at the time it didn't seem worth changing.